src/sentry/static/sentry/app/bootstrap/initializeSdk.tsx

```diff
@@ -0,0 +1,84 @@
+import * as Router from 'react-router';
+import {ExtraErrorData} from '@sentry/integrations';
+import * as Sentry from '@sentry/react';
+import SentryRRWeb from '@sentry/rrweb';
+import {Integrations} from '@sentry/tracing';
+
+import {DISABLE_RR_WEB, SPA_DSN} from 'app/constants';
+import {Config} from 'app/types';
+import {init as initApiSentryClient} from 'app/utils/apiSentryClient';
+
+/**
+ * We accept a routes argument here because importing `app/routes`
+ * is expensive in regards to bundle size. Some entrypoints may opt to forgo
+ * having routing instrumentation in order to have a smaller bundle size.
+ * (e.g.  `app/views/integrationPipeline`)
+ */
+function getSentryIntegrations(hasReplays: boolean = false, routes?: Function) {
+  const integrations = [
+    new ExtraErrorData({
+      // 6 is arbitrary, seems like a nice number
+      depth: 6,
+    }),
+    new Integrations.BrowserTracing({
+      ...(typeof routes === 'function'
```

Comment:
Can `routingInstrumentation` just be set to `undefined`? Might be slightly cleaner

```tsx
new Integrations.BrowserTracing({
	routingInstrumentation: routes !== 'function'
		? undefined
		: Sentry.reactRouterV3Instrumentation(
	        Router.browserHistory as any,
	        Router.createRoutes(routes()),
	        Router.match
	      )
});
```

Reply:
It can't

Reply:
😞